sound/soc/sof/intel/Kconfig

```diff
@@ -61,6 +61,15 @@ config SND_SOC_SOF_INTEL_DISABLE_ATOM_SST_MERRIFIELD
 	  and UCM files.
 	  If unsure select N.
 
+config SND_SOC_SOF_INTEL_BUILDTIME_DISABLE_SKL_SST_DRIVER
+	bool "Build-time exclusion with the Skylake/SST driver"
+	help
+	  This option relies on build-time mutual exclusion with the
```

Comment:
"relies" ... hmm, or should this be "This option provides build-time option for mutual exclusion"...?

Reply:
yes